game.ts

```diff
@@ -58,6 +58,10 @@ export async function fish(user: DiscordInteractionMemberUser) {
   };
 }
 
+export async function fishDuel(user: DiscordInteractionMemberUser, opponent: DiscordInteractionMember, wager:BigInteger) {

```

Comment:
I believe `user`, `opponent`, will have to be String type - at this point they are just the text from the command options.

`wager` can just be a simple Number type. ref - https://www.typescriptlang.org/docs/handbook/basic-types.html#number